cli/cli.ts

```diff
@@ -6136,6 +6136,7 @@ function internalCacheUsedBlocksAsync(): Promise<Map<pxt.BuiltTutorialInfo>> {
                                 if (!builtInfo.snippetBlocks[snippetHash]) builtInfo.snippetBlocks[snippetHash] = {};
                                 builtInfo.snippetBlocks[snippetHash][m1] = 1;
                                 builtInfo.usedBlocks[m1] = 1;
+                                //TODO: Fill builtInfo.HighlightedBlocks
```

Comment:
Created https://github.com/microsoft/pxt-arcade/issues/5516 to track

Reply:
for reference for other reviewers -- this cli fn fills out the tutorial info cache, which is used to cache used blocks / etc so first load of our 'hero' tutorials is faster. The reason it's okay that we're not fixing it in this pr is because currently micro:bit doesn't use this (no `cacheusedblocksdir` in that pxtarget) and while arcade tries to use this feature, it crashes at build time and doesn't produce anything (you can see `pxt.appTarget.tutorialInfo` is empty) -- minecraft does so we'll have to fix this eventually (and should fix for arcade too) but fine for now~